README.md

```diff
@@ -10,6 +10,11 @@
 
 ![freesound-player's appearance after selecting a sound as of 7:55 PM PST Tuesday, September 22, 2020](screenshots/soundpage-2020-09-25-at-8.02.03-PM-PST.png)
 
+## Related Repositories
+[`freesound-client`](https://github.com/amilajack/freesound-client)
```

Comment:
We should make these a list with markdown:

```
* [foo](bar)
* [foo](bar)
```

Reply:
Okay. Will do!